packages/react-codemods/transforms/pf3-pf4.button.test.js

```diff
@@ -3,25 +3,9 @@ import prettier from 'prettier';
 import { defineInlineTest } from 'jscodeshift/dist/testUtils';
 import transform from './pf3-pf4';
 
-/**
- * Codemod outputs should follow the EOL pattern of the target codebase, not
- * Patternfly's. Patternfly currently enforces LF as line ending, independent of
- * the OS building the codebase.
- *
- * JSCodeShift produces OS-dependent line endings however, LF for Unix based
- * systems, and CRLF for Windows based ones. This is also likely what most
- * projects would like to see the codemod prduce.
- *
- * To make sure we both adhere to Patternfly's conventions, and compare the
- * correct OS-specific line endings during the test cases' assertions, we store
- * expected values with LF line endings, and convert them into the OS-specific
- * ones at runtime using prettier.
- */
-const PRETTIER_EOL = SYSTEM_EOL === '\r\n' ? 'crlf' : 'cr';
 const prettierConfig = {
   ...prettier.resolveConfig.sync(process.cwd()),
-  parser: 'babel',
-  endOfLine: PRETTIER_EOL
```

Comment:
Will this work under Windows? I think this was added to support that
https://github.com/patternfly/patternfly-react/pull/1255/files#r251540310

Reply:
I don't think our project even builds on Windows currently due to path resolution problems. I'd love to support it in the future and fix things like #478

Reply:
To answer the question, as long as a Windows machine doesn't convert the line-endings to CRLF in JSCodemod, it will still work.

Reply:
Okay, not too long ago there was some effort made to support it but I guess without a Windows environment it is difficult to keep this
https://github.com/patternfly/patternfly-react/pull/1248
https://github.com/patternfly/patternfly-react/pull/1251